src/Appwrite/Utopia/Database/Validator/Query/Select.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Query;
+
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+
+class Select extends Base
+{
+    protected array $schema = [];
+
+    /**
+     * Query constructor
+     *
+     */
+    public function __construct(array $attributes = [])
+    {
+        foreach ($attributes as $attribute) {
+            $this->schema[$attribute->getAttribute('key')] = $attribute->getArrayCopy();
+        }
+    }
+
+    /**
+     * Is valid.
+     *
+     * Returns true if method is TYPE_SELECT selections are valid
+     *
+     * Otherwise, returns false
+     *
+     * @param $query
+     * @return bool
+     */
+    public function isValid($query): bool
+    {
+        /* @var $query Query */
+
+        if ($query->getMethod() !== Query::TYPE_SELECT) {
```

Comment:
Are there any attributes we do not want to be queried like `_uid` and stuff? It is already handled by the response models but do we want to enforce it here as well ?

Reply:
Selecting "_uid" is blocked by Utopia I Just checked.